org.eclipse.jdt.debug.tests/tests/org/eclipse/jdt/debug/tests/core/LineTrackerTests.java

```diff
@@ -181,6 +181,14 @@ public void testHyperLink() throws Exception {
 	    try {
 			ConsoleLineTracker.setDelegate(this);
 			fTarget = launchAndTerminate("ThrowsNPE");
+			long startTime = System.currentTimeMillis();
```

Comment:
Please use nanoTime() instead as TimeMillis is not guaranteed to be monotonic at leap seconds.

Reply:
okay 👍